packages/core/src/button/base-button.element.scss

```diff
@@ -135,16 +113,25 @@
 
 :host([action='outline']) {
   --background: #{$cds-alias-object-opacity-0};
-  --border-color: var(--color);
+  --background-hover: #{$cds-alias-status-info-tint};
   --color: #{$cds-alias-status-info};
+  --border-color: var(--color);
+}
+
+:host([action='outline']:hover),
+:host([action='outline']:focus),
+:host([action='outline']:active) {
+  --background: var(--background-hover);
 }
 
 :host([status='success'][action='outline']) {
   --color: #{$cds-alias-status-success};
+  --background-hover: #{$cds-alias-status-success-tint};
 }
 
 :host([status='danger'][action='outline']) {
   --color: #{$cds-alias-status-danger};
+  --background-hover: #{$cds-alias-status-danger-tint};
```

Comment:
Missing hover states for outline buttons.

Reply:
Is there no hover color for the inverse outline button? Or does the default for outline (`$cds-alias-status-info-tint`) work for inverse?

Reply:
Good catch, fixed!